scalding-core/src/main/scala/com/twitter/scalding/StringUtility.scala

```diff
@@ -0,0 +1,39 @@
+package com.twitter.scalding
+
+object StringUtility {
```

Comment:
the std. dev is probably huge due to occasionally hitting GC. Properly benchmarking is a bit of an art. There are some packages to make it easier, but this is fine for now.